index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <source src="main.js"></sourc>
```

Comment:
This should be <script src="javascript.js"></script> 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script